SPECS/msgpack/msgpack.spec

```diff
@@ -0,0 +1,57 @@
+Summary:        MessagePack implementation for C and C++
+Name:           msgpack
+Version:        3.2.1
+Release:        2%{?dist}
+License:        Boost
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://msgpack.org
+#Source0:       https://github.com/%{name}/%{name}-c/archive/cpp-%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
```

Comment:
Something's not right either here or in the URL above. The file downloaded using that URL is `msgpack-c-cpp-3.2.1.tar.gz`. We should use the downloaded file's name to avoid any confusion in case someone wants to update later or just verify this package.

Reply:
Agree- this was a weird case where the upstream GitHub is managed oddly, and the maintainers release the combined sources of two branches under one file. I have verified that `msgpack-c-cpp-3.2.1.tar.gz` is the correct file to be pulling, and I'll rename this to use the downloaded filename.

Reply:
Source reuploaded and name changed here. Other comments address in this pass. Thanks!